### DISCUSSION TITLE
Blocking WAN

### DISCUSSION THREAD

**Discussion by phantum29** [mrlt8/docker-wyze-bridge#185] [Q&A]
Hello, after setup of everything, is it possible to block WAN access to both the container and the wyze cams. I have an isolated vlan for IOT, and I'd like to disable WAN on that vlan, as it's on to enable setup of the wyze cams.

**Comment by mrlt8** [comment]
No. Unfortunately, it seems like the wyzecams themselves cannot startup without WAN access.

It may be possible to have the cam connect to the internet once at startup and then block WAN access, but it could cause issues as the cams fail to phone home.

https://github.com/mrlt8/docker-wyze-bridge/issues/121
